tableauserverclient/server/request_factory.py

```diff
@@ -469,6 +469,90 @@ def update_req(self, site_item):
+        if site_item.web_extraction_enabled is not None:
+            site_element.attrib['webExtractionEnabled'] = str(site_item.web_extraction_enabled).lower()
+        if site_item.run_now_enabled is not None:
+            site_element.attrib['runNowEnabled'] = str(site_item.run_now_enabled).lower()
+        if site_item.metrics_content_type_enabled is not None:
+            site_element.attrib['metricsContentTypeEnabled'] = str(site_item.metrics_content_type_enabled).lower()
+        if site_item.notify_site_admins_on_throttle is not None:
+            site_element.attrib['notifySiteAdminsOnThrottle'] = str(site_item.notify_site_admins_on_throttle).lower()
+        if site_item.authoring_enabled is not None:
+            site_element.attrib['authoringEnabled'] = str(site_item.authoring_enabled).lower()
+        if site_item.custom_subscription_email_enabled is not None:
+            site_element.attrib['customSubscriptionEmailEnabled'] = \
+                str(site_item.custom_subscription_email_enabled).lower()
+        if site_item.custom_subscription_email is not None:
+            site_element.attrib['customSubscriptionEmail'] = str(site_item.custom_subscription_email).lower()
+        if site_item.custom_subscription_footer_enabled is not None:
+            site_element.attrib['customSubscriptionFooterEnabled'] =\
+                str(site_item.custom_subscription_footer_enabled).lower()
+        if site_item.custom_subscription_footer is not None:
+            site_element.attrib['customSubscriptionFooter'] = str(site_item.custom_subscription_footer).lower()
+        if site_item.ask_data_mode is not None:
+            site_element.attrib['askDataMode'] = str(site_item.ask_data_mode).lower()
+        if site_item.named_sharing_enabled is not None:
+            site_element.attrib['namedSharingEnabled'] = str(site_item.named_sharing_enabled).lower()
+        if site_item.mobile_biometrics_enabled is not None:
+            site_element.attrib['mobileBiometricsEnabled'] = str(site_item.mobile_biometrics_enabled).lower()
+        if site_item.sheet_image_enabled is not None:
+            site_element.attrib['sheetImageEnabled'] = str(site_item.sheet_image_enabled).lower()
+        if site_item.cataloging_enabled is not None:
+            site_element.attrib['catalogingEnabled'] = str(site_item.cataloging_enabled).lower()
+        if site_item.derived_permissions_enabled is not None:
+            site_element.attrib['derivedPermissionsEnabled'] = str(site_item.derived_permissions_enabled).lower()
+        if site_item.user_visibility_mode is not None:
+            site_element.attrib['userVisibilityMode'] = str(site_item.user_visibility_mode).lower()
+        if site_item.use_default_time_zone is not None:
+            site_element.attrib['useDefaultTimeZone'] = str(site_item.use_default_time_zone).lower()
+        if site_item.time_zone is not None:
+            site_element.attrib['timeZone'] = str(site_item.time_zone).lower()
```

Comment:
timeZone is not a boolean, and I believe it is case-sensitive. The .lower() should be removed.
Same applies in create_req().